python/llm/test/inference/test_transformers_api.py

```diff
@@ -109,5 +111,60 @@ def test_transformers_chatglm_for_causallm(self):
         res = 'Paris' in output_str        
         self.assertTrue(res)
 
+@pytest.mark.parametrize('prompt, answer', [
+    ('What is the capital of France?\n\n', 'Paris')
+    ])
+@pytest.mark.parametrize('Model, Tokenizer, model_path',[
+    (AutoModel, AutoTokenizer, os.environ.get('ORIGINAL_CHATGLM2_6B_PATH')),
```

Comment:
shall we also add `load_low_bit` UT for Bloom / Llama ?

Reply:
Thanks for the comments. We could add them in further PRs.